src/lib/components/NewLayeredMap/components/MousePosition.js

```diff
@@ -51,9 +51,10 @@ const MousePosition = (props) => {
                 return latlng;
             },
     
-            updateHTML: function(x, y, z) { // TODO: add default measurement unit, make it passable with props
+            updateHTML: function(x, y, z, zNotZero) { // TODO: add default measurement unit, make it passable with props
+                const z_string = zNotZero? " z: " + z + "m" : "";
                 this._latlng.innerHTML ="<span style = 'background-color: #ffffff; border: 2px solid #ccc; padding:3px; border-radius: 5px;'>"
-                                       + "x: " + x + "m y: " + y + "m z: " + z  + "m" +"</span>"
+                                       + "x: " + x + "m y: " + y + "m"  + z_string +"</span>"
```

Comment:
Isn't this something that should be specified by _focusedImageLayer.options.unit_ or something? 🤔